5.x/crud-fields.md

```diff
@@ -1635,6 +1635,43 @@ Input preview:
 
 > NOTE: if you are having trouble uploading big images, please check your php extensions **apcu** and/or **opcache**, users have reported some issues with these extensions when trying to upload very big images. REFS: https://github.com/Laravel-Backpack/CRUD/issues/3457
 
+<hr>
+
+<a name="phone"></a>
+### phone <span class="badge badge-pill badge-info">PRO</span>
+
+Show a telephone input and let the user choose a flag from dropdown.
+
+```php
+[   // phone
+    'name'  => 'phone', // db column for phone
+    'label' => 'Phone',
+    'type'  => 'phone',
+
+    // OPTIONALS
+    // options offically supported and tested, this options will be prioritized to config parameters
+    // this mean if you send parameter "initialCountry" => "us" 
+    // and "config" => ["initialCountry" => "us"], "cl" value will be used
```

Comment:
I'd move this note AFTER the "officially supported" options. They only need to know about that if they wanna go deeper, so that's a better moment to say it I think, and we introduce them to `config` then.

So I'd rephrase to:

_the above are some officially supported and tested options, provided by intlTelInput.js; there are more, we just haven't tested them; check out their docs - you can use any of them inside the config attribute; take note that the directly defined attributes will override anything defined in the config;_